Update version range for Liberty install to newer version

The current version range is specified as `[22.0.0.3,)` which is quite old. A user recently reported seeing this error when trying to create a server.

```
[ERROR] Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.11.2:install-server (prepare-package) on project dv-caching: Could not get the highest version from the range: [22.0.0.3,): Unable to resolve version range from io.openliberty:openliberty-kernel:zip:[22.0.0.3,) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.11.2:install-server (prepare-package) on project dv-caching: Could not get the highest version from the range: [22.0.0.3,)
```

It would be good to update the first version to something recent like 25.0.0.2 (or 25.0.0.3 which is scheduled to GA next week).

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -22,7 +22,7 @@ jobs:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
         RUNTIME: [ol, wlp]
-        RUNTIME_VERSION: [24.0.0.12]
+        RUNTIME_VERSION: [25.0.0.3]
         java: [21, 17, 11, 8]
         exclude:
         - java: 8
@@ -80,7 +80,7 @@ jobs:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
         RUNTIME: [ol, wlp]
-        RUNTIME_VERSION: [24.0.0.12]
+        RUNTIME_VERSION: [25.0.0.3]
         java: [21, 17, 11, 8]
         exclude:
         - java: 8

--- a/README.md
+++ b/README.md
@@ -109,7 +109,7 @@ Example using the `runtimeArtifact` parameter to install a WebSphere Liberty run
         <runtimeArtifact>
             <groupId>com.ibm.websphere.appserver.runtime</groupId>
             <artifactId>wlp-webProfile8</artifactId>
-            <version>23.0.0.9</version>
+            <version>24.0.0.9</version>
             <type>zip</type>
         </runtimeArtifact>
     </configuration>

--- a/docs/installation-configuration.md
+++ b/docs/installation-configuration.md
@@ -39,7 +39,7 @@ Example for using the `runtimeArtifact` parameter:
         <runtimeArtifact>
             <groupId>com.ibm.websphere.appserver.runtime</groupId>
             <artifactId>wlp-webProfile8</artifactId>
-            <version>22.0.0.3</version>
+            <version>25.0.0.3</version>
             <type>zip</type>
         </runtimeArtifact>
     </configuration>
@@ -119,7 +119,7 @@ The `runtimeArtifact` configuration is very flexible. It includes the use of pro
 * Use the one specified in `runtimeArtifact`.
 * Use the version from a matching `dependency`, if found, for the calculated `groupId:artifactId` from steps 1 and 2.
 * Use the version from a matching `dependencyManagement`, if found, for the calculated `groupId:artifactId` from steps 1 and 2.
-* Default to the latest available using a range like `[22.0.0.3,)`.
+* Default to the latest available using a range like `[24.0.0.3,)`.
 
 ##### Example
 
@@ -131,7 +131,7 @@ Example "resolving" the `runtimeArtifact` version from matching `dependencyManag
       <dependency>
         <groupId>io.openliberty</groupId>
         <artifactId>openliberty-runtime</artifactId>
-        <version>22.0.0.2</version>
+        <version>25.0.0.3</version>
         <type>zip</type>
       </dependency>
     </dependencies>

--- a/docs/installation-configuration.md
+++ b/docs/installation-configuration.md
@@ -119,7 +119,7 @@ The `runtimeArtifact` configuration is very flexible. It includes the use of pro
 * Use the one specified in `runtimeArtifact`.
 * Use the version from a matching `dependency`, if found, for the calculated `groupId:artifactId` from steps 1 and 2.
 * Use the version from a matching `dependencyManagement`, if found, for the calculated `groupId:artifactId` from steps 1 and 2.
-* Default to the latest available using a range like `[24.0.0.3,)`.
+* Default to the latest available using a range like `[25.0.0.3,)`.
 
 ##### Example
 

--- a/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java
+++ b/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java
@@ -281,8 +281,8 @@ public abstract class BasicSupport extends AbstractLibertySupport {
                 Artifact artifact = getResolvedArtifact(assemblyArtifact);
 
                 if (artifact == null) {
-                    initLog.debug("Defaulting runtimeArtifact version to '[22.0.0.3,)'");
-                    assemblyArtifact.setVersion("[22.0.0.3,)");
+                    initLog.debug("Defaulting runtimeArtifact version to '[25.0.0.3,)'");
+                    assemblyArtifact.setVersion("[25.0.0.3,)");
                     artifact = createArtifact(assemblyArtifact);
                 }
                 

